modules/client-configuration/pages/registration-bootstrap.adoc

```diff
@@ -140,10 +140,11 @@ cd /srv/www/htdocs/pub/bootstrap/
 . Run this command to execute the bootstrap script on the client; replace [systemitem]`` EXAMPLE.COM`` with the host name of your client:
 +
 ----
-cat bootstrap-sles12.sh | ssh root@EXAMPLE.COM /bin/bash
+cat bootstrap-sles15.sh | ssh root@EXAMPLE.COM /bin/bash
 ----
 The script downloads the required dependencies located in the repositories directory you created earlier.
 . When the script has finished running, you can check that your client is registered correctly by opening the {productname} {webui} and navigating to menu:Systems[Overview] to ensure the new client is listed.
+. If you used the script to register Salt client, you can open {productname} {webui} and navigate to menu:Salt[Keys] where you can accept the client key.
```

Comment:
Wondering whether "Salt client" requires an article?

Reply:
We will have it in different PR.